include/sdf/JointAxis.hh

```diff
@@ -197,13 +197,13 @@ namespace sdf
     public: double Effort() const;
 
     /// \brief Set the value for enforcing the maximum joint effort applied.
-    /// Limit is not enforced if value is negative.
+    /// Limit is not enforced if value is infinity.
     /// \param[in] _effort Effort limit.
     /// \sa double Effort() const
     public: void SetEffort(double _effort);
 
     /// \brief Get the value for enforcing the maximum joint velocity. The
-    /// default value is -1.
+    /// default value is infinity.
```

Comment:
Maybe it's worth mentioning in the docs that these values are absolute, so they apply to both positive and negative directions?

Reply:
sure, I'll add it

Reply:
0227719